src/components/AddressSearch.js

```diff
@@ -128,6 +139,15 @@ const AddressSearch = (props) => {
             country: '',
         };
 
+        /**
+         * The short name for states in USA is the state's ISO code, which is needed in the StatePicker.
+         * If the country is outside USA, instead of showing the short code of the state, we want to show
+         * the full name of the state.
+         */
```

Comment:
Similarly, here's what I'd probably say here - let me know what you think:

```suggestion
        // If the address is in the US, use the full length state name since we're displaying the address's
        // state / province in a TextInput instead of in a picker.
```

Reply:
TRUE good catch sorry :D